shared/js/background/trackers.es6.js

```diff
@@ -198,6 +170,8 @@ function checkTrackersWithParentCompany (url, siteDomain, request) {
             tracker.rules.some(ruleObj => {
                 if (requestMatchesRule(request, ruleObj.rule) && matchRuleOptions(ruleObj, request, siteDomain)) {
                     toBlock.rule = ruleObj
+                    toBlock.trackerData = tracker
```

Comment:
Can we immediately check the whitelist here and bail out if request is whitelisted? Wouldn't it save us some work?

Reply:
👍 good idea.